store/attribute/actions.js

```diff
@@ -2,58 +2,68 @@
  * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
  * See LICENSE for license details.
  */
+import { types } from './mutations';
```

Comment:
It might be confusing, importing sth from mutations, maybe it would be better to keep constants values in other file ?

Reply:
Good point, I will pull out them to separated file 'mutation-types.js' in each module according to official vuex docs:
https://vuex.vuejs.org/guide/mutations.html